src/test/java/com/todo/Hiplanner/Memo/MemoTest.java

```diff
@@ -91,4 +91,21 @@ public void deleteMemo(){
 
 
     }
+
+
+    // 메모 내용 변경 테스트
+    @Test
+    public void updateMemo(){
+        Memo memo = new Memo();
+        memo.setId("wodud");
+        memo.setTitle("기능구현");
+        memo.setContent("메모 수정");
+        memo.setPlanno(2);
+        plannerMapper.updateMemo(memo);
+
+        Memo memoDetail = plannerMapper.getMemoDetail(memo);
```

Comment:
```suggestion
        // when
        Memo memoDetail = plannerMapper.getMemoDetail(memo);
```

시간되시면 한 번 검색해서 보시는 것도 좋아요!
테스트코드를 짤 때, given-when-then 패턴으로 설계하며, (개인의 자유지만) 주석으로 각 단계를 표시해두기도 합니다 ㅎㅎ

Reply:
given-when-then 전에도 말씀해주신 거 같은데 제가 메모하는 습관을 길러서 다음번에는 잊지 않고 꼭 해보겠습니다.